spec/support/features/session_helpers.rb

```diff
@@ -6,6 +6,7 @@ def login_as(user)
       user.reload # because the user isn't re-queried via Warden
       super(user, scope: :user, run_callbacks: false)
     end
+
```

Comment:
Super minor, but you could back out this whitespace change if you're making other changes.